config/i18n.json

```diff
@@ -528,14 +529,14 @@
     "ExcludeItem": "Exclude Item",
     "Filter": "Filters",
     "GeneratedBuilds": "Builds",
-    "LimitedCombos": "You have {{combosWithoutCaps, number}} possible stat combinations, but we only looked at {{combos, number}} of them to keep DIM from crashing. As a result, some possibilities are not shown. Lock some items or perks or dismantle some old armor to get under the limit.",
+    "LimitedCombos": "You have {{combosWithoutCaps, number}} possible item combinations, but we only looked at {{combos, number}} of them to keep DIM from crashing. As a result, some possibilities are not shown. Lock some items or perks or dismantle some old armor to get under the limit.",
     "LockEquipped": "Equipped",
     "LockItem": "Lock item",
     "LockPerk": "Select energy or perk",
     "NewEmptyLoadout": "Create Empty Loadout",
     "NoBuildsFound": "No builds found. Try expanding your filtering criteria.",
-    "NumStatMixes": "{{count, number}} stat mix",
-    "NumStatMixes_plural": "{{count, number}} stat mixes",
+    "NumBuilds": "{{count, number}}",
+    "NumBuilds_plural": "{{count, number}}",
```

Comment:
If you do not have any words, these are effectively the same, and can be consolidated down to `"NumBuilds": "{{number}}",`

Reply:
Cool, I was messing around with it but it was generating the plural version which I now assume was due to the count parameter being passed in. I will look up the i18next docs today and sort it out.

Reply:
@delphiactual, is this change what you meant? 

I did a bit of digging to see if I could use i18n for the number localisation without the need of a key but couldn't figure out a way. The reason I wanted to use i18n is normally I try to keep formatting coming from a single source so that consistency is easier to achieve.